crates/air/src/commands/lsp.rs

```diff
@@ -4,7 +4,7 @@ use crate::ExitStatus;
 #[tokio::main]
 pub(crate) async fn lsp(_command: LspCommand) -> anyhow::Result<ExitStatus> {
     // Returns after shutdown
-    lsp::start_lsp(tokio::io::stdin(), tokio::io::stdout()).await;
+    lsp::start_server(tokio::io::stdin(), tokio::io::stdout()).await;
```

Comment:
Happy to rename this back but i got a little confused seeing `TestClient` accept a lambda function that called `start_lsp()`. I thought `start_lsp()` did something client related when I saw that.